src/dash-playlist-loader.js

```diff
@@ -86,6 +86,19 @@ const dashPlaylistUnchanged = function(a, b) {
   return true;
 };
 
+/**
+ * Use the representation IDs from the mpd object to create groupIDs, this allows for continuous playout across periods
+ * with the same representation IDs (continuous periods as defined in DASH-IF 3.2.12). This is assumed in the mpd-parser
+ * as well. If we want to support periods without continuous playback this function may need modification as well as the
+ * parser.
+ */
+const dashGroupId = (type, group, label, playlist) => {
+  // NAME is set to the id in the parser.
+  const playlistId = playlist.attributes && playlist.attributes.NAME ? playlist.attributes.NAME : label;
```

Comment:
nit: You can save a bit of code by using the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) operator here, and then using an OR statement:
`const playlistId = playlist.attributes?.NAME || label;`

Reply:
I think the question is whether the babel config transpiles optional chaining, because its output is a lot larger comparably.

Reply:
Good point, if this is going to cause transpiling issues then it's definitely not worth the change. However, I would expect it to be able to transpile optional chaining.

Reply:
@wseymour15 I love that syntax change, however it does unfortunately transpile to a significantly longer switch statement in this case.
` const playlistId = ((_playlist$attributes = playlist.attributes) == null ? void 0 : _playlist$attributes.NAME) || label;`

Reply:
That said, I think I can actually get away with `const playlistId = playlist.attributes.NAME || label;` per the NAME being the representation `id` from the DASH parser which is a required, unique property. My old typescript habits leaking through with that extra check that `playlist.attributes` exists